addons/compat_ws/compat_ws_realisticweights/config.cpp

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class SUBADDON {
+        name = COMPONENT_NAME;
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {
+            "data_f_lxWS_Loadorder"
```

Comment:
not strictly needed, but I don't see any harm in adding this just in-case someone doesn't want it?
```suggestion
            "data_f_lxWS_Loadorder", "ace_realisticweights"
```